### DISCUSSION TITLE
Oled displays?

### DISCUSSION THREAD

**Discussion by amazingslothrider** [nickcoutsos/keymap-editor#174] [Q&A] [answered]
Hi everyone

First off, thank you Nick for making this keymap editor. I'm a total n00b and feel very out of my depth with coding. I was totally lost until I stumbled onto this and without it I don't think I'd have been using the Corne at all. So sincerely, thank you. 

I've got a question about oled displays. I've tried the readme but couldn't find any information on it. Since flashing my corne with the firmware I'd made here, I have lost the displays. Is there anyway to get them back or change what is on them using the editor? Ideally I want to see the battery life, what layer I'm on and what device I'm connected to as I keep getting 'lost'.

If I'm totally on the wrong track could someone please help me to find out how to get the displays working again?

Cheers

Fuzzy

**Comment by nickcoutsos** [comment, marked as answer]
Hi @amazingslothrider!

It doesn't sound like your displays are broken. At least, the keymap editor doesn't affect them in any way -- there's nothing in the app to manage any display related configuration or underlying ZMK code.

I'm looking at [your zmk-config repository](https://github.com/amazingslothrider/zmk-config) which is the keyboard in question, I think? It looks like the display hasn't been enabled in that firmware from the very beginning, I'm wondering if maybe this is something that came pre-flashed.

OLED displays are kind of out of my wheelhouse, I don't have access to any myself, but I see that
[`CONFIG_ZMK_DISPLAY`](https://github.com/amazingslothrider/zmk-config/blob/main/config/corne.conf#L6) is not enabled in your corne config. I'd start by un-commenting that line (remove the `# ` (don't forget that space) from the beginning of the highlighted line) and go from there. If you're lucky that might be it, but if there's more involved it may be worth asking on ZMK's Discord server.

**Reply by amazingslothrider** [reply]
Nick, thank you once again! Your instructions were nice and simple and it was so much easier than I anticipated to tweak the code (I thought I had to go through terminal etc). Now the oled's are working, I can see what I'm connected to, and what layer I'm on! 

When I got the Corne it had been pre-flashed, so when I set it up, I lost that setting. Such a small thing but having the displays really helps. If anything, it's giving me more confidence to dive deeper into what I can do with ZMK and your keymap editor, as I know I won't get 'lost' thanks to the displays.

**Reply by nickcoutsos** [reply]
Great news, glad you're back in business :)
